assets/scripts/app.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const game = require('./game.js')
```

Comment:
You can use better "import... from..." instead of "const... = require(...)" from ES6

Reply:
Oh? I thought that was only in React.js?

Reply:
It's the Javascript ES6 :) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import